bin/ffscreencast

```diff
@@ -273,6 +306,8 @@ print_help() {
 	echo "                  Specify additional ffmpeg arguments for the output encoding."
 	echo "                  Use: --oargs=\"-crf 0\""
 	echo "                  Default: '-crf 0 -preset ultrafast'"
+	echo 
+	echo "-o           Output video file name"
```

Comment:
I think this option should document its usage, similar to how the others do.

Reply:
Yeah, Today itself this pull request will be merged/rejected.